pennylane_lightning/src/gates/cpu_kernels/avx_common/ApplyPhaseShift.hpp

```diff
@@ -33,10 +34,13 @@ template <typename PrecisionT, size_t packed_size> struct ApplyPhaseShift {
 
     constexpr static size_t packed_size_ = packed_size;
 
-    static constexpr auto createPermutation(size_t rev_wire) {
-        std::array<uint8_t, packed_size> perm = {
-            0,
-        };
+    /**
+     * @brief Permutation for applying `i` if a bit is 1
+     *
+     * FIXME: clang++-12 currently does not accept consteval here.
```

Comment:
I guess we can keep this until we update the manylinux2014 wheels.

Reply:
:+1: